src/server/data.js

```diff
@@ -4,13 +4,13 @@ module.exports = {
 
 function getPeople() {
   return [
-    { id: 1, firstName: 'John', lastName: 'Papa', age: 25, location: 'Florida' },
```

Comment:
don't commit changes on files that we do not need :) only proper change is deletion :)